spring-xd-shell/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,8 @@
+log4j.rootLogger=INFO, stdout
```

Comment:
This file was added to avoid a conflict from kafka tests jar's log4j.properties. As you can notice the upstream master test reports would use kafka tests jar's log4j and I believe it is set to WARN level.